src/LaunchDarkly/LDClient.php

```diff
@@ -75,63 +94,63 @@ public function getFlag($key, $user, $default = false) {
         return $this->toggle($key, $user, $default);
     }
 
-   /** 
-    * Calculates the value of a feature flag for a given user.
-    *
-    * @param string  $key     The unique key for the feature flag
-    * @param LDUser  $user    The end user requesting the flag
-    * @param boolean $default The default value of the flag
-    *
-    * @return boolean Whether or not the flag should be enabled, or `default` if the flag is disabled in the LaunchDarkly control panel
-    */
+    /**
+     * Calculates the value of a feature flag for a given user.
+     *
+     * @param string $key The unique key for the feature flag
+     * @param LDUser $user The end user requesting the flag
+     * @param boolean $default The default value of the flag
+     *
+     * @return mixed Whether or not the flag should be enabled, or `default`
+     */
```

Comment:
I think we need to standardize on what the primary feature flag request methods will be for both the typed and untyped languages. My current thinking is that we should:

* Deprecate `toggle`, because the noun / verb confusion is problematic
* For untyped languages, have a function called `variation` that works for either bool or multivariates
* For typed languages have functions `typeVariation` that are strongly typed, *including* one for `boolVariation`